.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: rust
 cache: cargo
 rust:
-  - stable
-  - beta
   - nightly
```

Comment:
I think we should only use one nightly because the proc-macro doesn't change that much, also it would slow builds